src/report/report-list.html

```diff
@@ -97,5 +97,24 @@ <h2>{{'report.viewReports' | message}}</h2>
             </tbody>
         </table>
     </div>
+    <div class="openlmis-table-container">
+        <table>
+            <caption class="error" ng-show="!vm.reports.length && !vm.supersetReports">
+                {{'report.noReports' | message}}
```

Comment:
For optimalization load this translation only once by using ::

https://docs.angularjs.org/guide/expression#one-time-binding

`{{:: 'report.noReports' | message}}`

Reply:
sure thing, updated.